test/tests/utilities_internalTest.js

```diff
@@ -147,15 +147,20 @@ describe("Zotero.Utilities.Internal", function () {
 		});
 		
 		it("should extract PMID", async function () {
-			var id = "24297125";
-			var identifiers = ZUI.extractIdentifiers(id);
-			assert.lengthOf(identifiers, 1);
+			var identifiers = ZUI.extractIdentifiers("1 PMID:24297125,222 3-4 1234567890, 123456789");
+			assert.lengthOf(identifiers, 4);
 			assert.lengthOf(Object.keys(identifiers[0]), 1);
-			assert.propertyVal(identifiers[0], "PMID", id);
+			assert.lengthOf(Object.keys(identifiers[1]), 1);
+			assert.lengthOf(Object.keys(identifiers[2]), 1);
+			assert.lengthOf(Object.keys(identifiers[3]), 1);
+			assert.propertyVal(identifiers[0], "PMID", "1");
+			assert.propertyVal(identifiers[1], "PMID", "24297125");
+			assert.propertyVal(identifiers[2], "PMID", "222");
+			assert.propertyVal(identifiers[3], "PMID", "123456789");
```

Comment:
Why wouldn't we want to match the `1234567890` in the above?

Reply:
Because latest PMIDs still have 8 digits, so 9 digits are sufficient. No need to extract longer numbers.